### DISCUSSION TITLE
A Short Rant About Quiet Quitting (or, Why Heroism Can Be Toxic) – American Alliance of Museums

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#5669] [Q&A]
I thought this article might hit home with a lot of us.

[A Short Rant About Quiet Quitting (or, Why Heroism Can Be Toxic) – American Alliance of Museums](https://www.aam-us.org/2023/02/15/a-short-rant-about-quiet-quitting-or-why-heroism-can-be-toxic/) 

It got me wondering whether we are fostering `toxic heroism` in the Arctos Community and if we are, what can we do to mitigate it? I'd love to know how people feel about this article. Please add your thoughts and opinions!

**Comment by lin-fred** [comment]
This: 

"I’ve written before about the “[sacrifice measure](https://www.aam-us.org/2014/09/23/the-museum-sacrifice-measure/)”—the tendency of people coming to museum work to accept less money for a job that gives them the opportunity to do what they love. Cumulatively, this contributes to depressed wages in the industry and sets everyone up for disappointment, particularly workers who find the actual work does not align closely enough with their passion, or _face with the stark reality that their wages aren’t enough to pay the rent_."

It is definitely an expectation

**Reply by Jegelewicz** [reply]
Yes, I found so much of this article describing not only my experience, but the experiences shared with me by others. I wonder how we can support each other and create an environment where it isn't expected that one should be paid less or accept underemployment just to work in a museum? I recently read something to this effect `any job with an annual salary that wouldn't pay the annual cost of training for that position (read tuition) is not offering a realistic salary`. That doesn't even consider that tuition doesn't feed or house most people.

**Reply by ewommack** [reply]
I think a lot of this has implications also for DUI, and how much pressure is placed on groups to do more with less. Would it be ok to share the conversation with the committee too?

**Comment by dbloom** [comment]
This is such a great piece and our little slice of the museum community would do well to think about the issues raised here. The same community would benefit from the use of strategic foresight and the ways that it helps people to think ahead. Elizabeth Merritt does a great service for the art, history, and "front-of-house" sides of the museum world. Collections would benefit from the same types of analysis and thinking. So much more could be said about this and how non-PhDs are valued in the academic setting, too.

**Reply by ewommack** [reply]
Or any staff member. There is definitely a divide in some between staff and everyone else. Sometimes I feel like I'm running just to catch up with the wake left behind in the museum from some projects.

**Comment by ewommack** [comment]
I think where I have a hard time dividing out is I would do parts of my job as volunteer work (and have in other institutions), but now it is my job. So when I stay late to prep an object or install an accession, I know I would be doing this work as a volunteer in my off time at another institution because I would love and learn from it, but that just means in my job I'm not going home on time. It's hard to draw that line.

**Reply by Jegelewicz** [reply]
Sigh, I feel this so very much - and I also always think that if it needs to be done, someone should be paid to do it.

**Comment by campmlc** [comment]
Relevant to this: our cms have been putting together a graph which shows
that staff collection managers at our institution are the lowest paid of
all university employees in our grade, with no tiers for advancement, to
the point of being outliers. At the same time our job descriptions require
the highest level of education and training in the same grade.

**Reply by Jegelewicz** [reply]
Time for a union?
